<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="res">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>183</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="1" column="0">
     <widget class="QRadioButton" name="dif">
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="ann">
      <property name="text">
       <string>Аннуитетный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>330</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>20</y>
     <width>218</width>
     <height>371</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="total">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>342</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>311</y>
     <width>170</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата по кредиту, руб</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="overpayment">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>311</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>342</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата, руб</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>19</y>
     <width>170</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая сумма кредита, pуб</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_credit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>60</y>
     <width>137</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="monthRB">
      <property name="text">
       <string>Месяцы</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="yearRB">
      <property name="text">
       <string>Годы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
